1706978084484	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1706979675211	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1706979675211	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1706979675215	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1706979675217	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1706980898738	Sync.SyncScheduler	DEBUG	Woke from sleep.
1706980898741	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1706980898741	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1706980898742	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1706980900743	Sync.Service	DEBUG	User-Agent: Firefox/122.0 (Linux x86_64) FxSync/1.124.0.20240123151424.desktop
1706980900743	Sync.Service	INFO	Starting sync at 2024-02-03 20:21:40 in browser session a2NKRlS-X7Qc
1706980900743	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1706980900747	Sync.Status	INFO	Resetting Status.
1706980900747	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1706980900748	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1706980900748	FirefoxAccounts	DEBUG	getOAuthToken enter
1706980900748	FirefoxAccounts	TRACE	getCachedToken returning cached token
1706980900749	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1706980900749	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1706980900750	Sync.SyncAuthManager	INFO	Getting sync key
1706980900750	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1706980900750	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1706980900750	FirefoxAccounts	DEBUG	getOAuthToken enter
1706980900750	FirefoxAccounts	TRACE	getCachedToken returning cached token
1706980900750	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1706980900751	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1706980900751	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1706980900770	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1706980900771	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 479\"  data: no]"}
1706980900771	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1706980900772	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1706980900772	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1706980900772	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1706980900773	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1706980900775	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1706980900778	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/207020392/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1706980900778	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1706980900778	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1706980900778	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1706980900779	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1706980900779	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1706980900782	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
